### ISSUE TITLE
pointing_button with modifiers and extra (optional) keys

### ISSUE THREAD

**Issue by djmcgreal** [pqrs-org/Karabiner-Elements#2119]
I am trying to modify cmd+middle mouse to shift+ops+middle mouse, because there's a cross platform app that doesn't recognise the command key.

```
                    {
                        "description": "Command+M3",
                        "manipulators": [
                            {
                                "from": {
                                    "modifiers": {
                                        "mandatory": [
                                            "left_command"
                                        ],
                                        "optional": [
                                            "any"
                                        ]
                                    },
                                    "pointing_button": "button3"
                                },
                                "to": {
                                    "modifiers": [
                                        "left_option",
                                        "left_shift"
                                    ],
                                    "pointing_button": "button3"
                                },
                                "type": "basic"
                            }
                        ]
                    }
```

This works, except when there are other, non-modifier buttons pressed at the same time.

cmd+button3 ✅
cmd+ctrl-button3 ✅
cmd+t+button3 ❌

Is there a way to configure the manipulator to trigger even when other keys are pressed? Or is this a bug?

Thanks, Dan.

**Comment by tekezo** [comment]
Thank you for feedback.

It's a bug and fixed at v12.8.91 (latest beta version).
Please confirm the latest beta version.

How to upgrade Karabiner-Elements to the latest beta version:
<https://pqrs.org/osx/karabiner/document.html#check-for-updates>

**Comment by djmcgreal** [comment]
Wow! That was fast. I tested the 12.8.90 immediately before posting!
I can confirm it's fixed in .91 :)
Thanks!
